.gitignore

```diff
@@ -35,3 +35,6 @@ jspm_packages
 
 # Optional REPL history
 .node_repl_history
+
+# JSDOC Output directory
```

Comment:
If there is no `.npmignore`, NPM will ignore files from `.gitignore` by default.